apps/ops/tasks.py

```diff
@@ -92,14 +92,13 @@ def job_execution_task_activity_callback(self, execution_id, *args, **kwargs):
     )
 )
 def run_ops_job_execution(execution_id, **kwargs):
-    if not settings.SECURITY_COMMAND_EXECUTION:
-        return
     with tmp_to_root_org():
         execution = get_object_or_none(JobExecution, id=execution_id)
-
     if not execution:
         logger.error("Did not get the execution: {}".format(execution_id))
         return
+    if not settings.SECURITY_COMMAND_EXECUTION and execution.job.type != Types.upload_file:
+        return
     _run_ops_job_execution(execution)
 
 
```

Comment:
Here are some of my observations on the provided code:

1. The `_run_operations_job_execution` function has a typo in its docstring ("operations", not "op") when it should be written as `JobExecution`. Fixing this would give the method's full name as `_run_ops_job_execution`.
   
2. The `_run_operations_command_task_activity_callback` function definition seems to have been copied from the existing `_run_operations_job_execution`, but there is currently no corresponding task for executing command tasks.

Regarding improvements, you can consider adding type annotations (i.e., `<class 'type'>`) at module/class/variable definitions to make the code more readable and maintainable (`from types import <type>`, etc.). Additionally, you might want to refactor `_run_operations_job_execution` so that its arguments do not require passing through `*args`, but rather expect them inside an object passed into the method (`get_object_or_none()` calls could use `object=None`). This will also allow users to easily pass their own instance of jobs where required.

Let me know if we need any further help!